### ISSUE TITLE
[Feat]: Helm chart not possible to reference

### ISSUE THREAD

**Issue by rickardp** [lowcoder-org/lowcoder#765]
**Is your feature request related to a problem? Please describe.**
It is possible that the existing Helm chart is not ready to be used directly, and the intent is for this to be copied. However, it would still be valuable to version this correctly and publish this to a registry. If it needs to be tweaked, consumers of this chart can use e.g. Kustomize to patch what is needed rather than having to sync and merge upstream changes to it.

**Describe the solution you'd like**
Suggestion is to:

Add a version to the chart that follows the tag
Publish the chart to e.g. ghcr.io so that it can be consumed as a Helm dependency

**Describe alternatives you've considered**
Git submodules could work, but is a clumsy and tedious method. Writing your own Helm chart from scratch, inspired by the existing one, which works but adds maintenance, also there may be subtle upstream changes requiring migrations which are not obvious.

**Additional context**

**Comment by voriol** [comment]
Maybe you could consider adding the helm chart to artifacthub

https://artifacthub.io/

**Comment by rickardp** [comment]
I would prefer if there was an official helm chart that I could refer to.

If I would do it myself I would upload it to a private registry (which is sort of what I do now)

**Comment by voriol** [comment]
Sorry @rickardp, maybe I didn't express myself well, I mean maybe the lowcoder team could consider uploading the helm chart to ArtifactHub. The process is simple and provides a lot of value.

**Comment by rickardp** [comment]
That makes more sense, I misunderstood "you" here.

Yes, that, or just the same GitHub ghcr.io where the Docker images go, if not wanting to maintain another account.

**Comment by ludomikula** [comment]
Yes, it is definitely something we will do. I will have a look into it shortly.

**Comment by voriol** [comment]
@ludomikula, if you need some help, please, ask! thanks

**Comment by Urriellu** [comment]
It looks that this has fallen out of priority? The [documentation](https://github.com/lowcoder-org/lowcoder/tree/main/deploy/helm) points to a Helm chart by Bitnami but that does not exist.
